octopoes/octopoes/xtdb/client.py

```diff
@@ -123,7 +123,7 @@ def query(self, query: str | Query, valid_time: datetime | None = None) -> list[
         res = self._session.post(
             f"{self.client_url()}/query",
             params={"valid-time": valid_time.isoformat()},
-            content=str(query),
+            content=" ".join(str(query).split()),
```

Comment:
Less traffic.

Reply:
Note that the function can _either_ take a `Query` or a `str` (and _is_ called with the latter). While Query seem to be properly formatted for strings this is not the case. Maybe we should make a patch to strictly only allow queries -- but if this is a point of major pain I am also happy to checkout the patch

Reply:
Ah so this helps in the cases where we still have the handwritten queries. Allowing strings is nice actually for debugging so I'd like to keep that

Reply:
I doubt it helps anything, sending a few extra spaces to XTDB shouldn't be a problem. I doubt this would be measurable at all, but if it was I think that creating a number of extra string objects in Python would be less efficient than sending a few extra spaces to XTDB.

In general I think we should make code as simple and easy to understand as possible. If there is no clear performance benefit, we should choose for simpler code and in this case that would mean not doing the split/join.